MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -55,15 +57,22 @@ def store_issue_details(issue, repo, db):
 	issue_details['created_at'] = issue['created_at']
 	issue_details['updated_at'] = issue['updated_at']
 	issue_details['labels'] = issue['labels']
-	
+
 	issue_details['issue_content_path'] = issue_content_path
 	# issue_details['test_output_path'] = test_output_path
 	issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+	issue_details['issue_quoted_jenkins_links'] = issue_quoted_jenkins_links
+    	issue_details['issue_quoted_test_names'] = issue_quoted_test_names
 	#Store issue content at issue_content_path
 	store_on_fs(issue['body'], issue_content_path)
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	store_jenkins_result = store_on_fs(extract_jenkins_link_and_testname(issue['body']), issue_quoted_jenkins_links)
```

Comment:
Same here, we only need to use `extract_jenkins_link_and_testname` to get `issue_quoted_jenkins_links` and `issue_quoted_test_names`, then save them in db with issue_details if they are not empty.